src/commands/circleci-continue-long-job-cancel.yml

```diff
@@ -11,14 +11,18 @@ parameters:
     description: the name of the environmental variable that contains the circleci token
     default: "CIRCLECI_TOKEN"
     type: string
+  ignore-jobs-with-name:
+    description: do not warn people about jobs with this name
```

Comment:
the text of this description is slightly different at every level. It looks like the python says it filters on it being a substring, whereas this looks like an exact match